tool/tsh/tsh.go

```diff
@@ -311,6 +322,10 @@ func Run(args []string) {
 	bench.Flag("duration", "Test duration").Default("1s").DurationVar(&cf.BenchDuration)
 	bench.Flag("rate", "Requests per second rate").Default("10").IntVar(&cf.BenchRate)
 	bench.Flag("interactive", "Create interactive SSH session").BoolVar(&cf.BenchInteractive)
+	bench.Flag("export", "Export the latency profile").BoolVar(&cf.BenchExport)
+	bench.Flag("path", "Path to save the latency profile to, default path is where .tsh directory is").StringVar(&cf.BenchExportPath)
```

Comment:
```suggestion
	bench.Flag("path", "Directory to save the latency profile to, default path is where .tsh directory is").StringVar(&cf.BenchExportPath)
```

Reply:
We usually use `.tsh` for local state data, like certificates and profiles. I think it's reasonable to use current directory as default.